.gitignore

```diff
@@ -177,3 +177,7 @@ node_modules
 # Asv stuff
 asv_bench/.asv/
 asv_bench/modin/
+

```

Comment:
I included this by mistake when I ran `git add -u`, should I make a separate PR for it? (it doesn't really correspond to any issue)

Reply:
I think this is ok